code/datums/mecha_designs.dm

```diff
@@ -0,0 +1,198 @@
+///////////////////////////////////
+//////////Mecha Module Disks///////
+///////////////////////////////////
+// Ripley
+/datum/design/ripley_main
+	name = "Exosuit Board (APLU \"Ripley\" Central Control module)"
+	desc = "Allows for the construction of a \"Ripley\" Central Control module."
+	id = "ripley_main"
+	req_tech = list("programming" = 2)
+	build_type = IMPRINTER
+	materials = list(MAT_GLASS = 1000)
+	build_path = /obj/item/circuitboard/mecha/ripley/main
+	category = list("Exosuit Modules")
+
+/datum/design/ripley_peri
```

Comment:
```suggestion
/datum/design/ripley_peripherals
```
Please avoid shortening words like this, you may now know what `peri` stands for, but future coders won't. See how the build path is similar: `/obj/item/circuitboard/mecha/ripley/peripherals`